easybuild/easyconfigs/a/artic-ncov2019/artic-ncov2019-2021.06.24-foss-2020b.eb

```diff
@@ -0,0 +1,93 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+# Updated to latest version 24. Jun 2021 and bumped to foss-2020b
+# to use it with pangolin
+# J. Sassmannshausen (GSTT/NHS UK)
+
+easyblock = 'PythonBundle'
+
+name = 'artic-ncov2019'
+local_commit = 'b3f2dda5e6d95bc1c5d95a04d4ef37d304479477'
+version = '2021.06.24'
+
+homepage = "https://github.com/artic-network/artic-ncov2019"
+description = """Initial implementation of an ARTIC bioinformatics platform 
+for nanopore sequencing of nCoV2019 novel coronavirus."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Biopython', '1.78'),
+    ('Pysam', '0.16.0.1'),
+    ('BWA', '0.7.17'),
```

Comment:
This software is x86 only and doesn't built e.g. on PPC due to unconditional dependency on x86 vector intrinsics.
Is this really required here?

Reply:
@Darkless012 How were those dependencies determined?

Reply:
I done a bit of digging and basically followed the install procedure for `artic` in `conda` like this:
```
conda install artic=1.2.1
conda install snakemake-minimal=5.8.1
conda install artic-network::rampart=1.2.0
```
As for `bwa` that gets installed. From what I understand, that is a problem on e.g. PPC. 
I done a bit of digging and I found that for ARM, but it also mentions PPC if I got that correctly:
https://github.com/lh3/bwa/issues/300
This is the link for ARM:
https://aws.amazon.com/de/blogs/publicsector/generalized-approach-benchmarking-genomics-workloads-cloud-bwa-read-aligner-graviton2/

I would suggest to leave `BWA` in the EasyConfig file, but add a comment so PPC and ARM know what they need to do. A probably better solution would be to get the BWA EC installation working on both ARM and PPC, but that is outside my set of skills and I don't have access to either PPC or ARM. 

I can provide you with a full list of what `conda` is installing offline if you want to have a look.

Reply:
Hm, BWA is not the only issue. minimap2-2.18-GCCcore-10.2.0.eb fails due to `gcc -c -g -Wall -O2 -Wc++-compat  -msse2 -mno-sse4.1 [...]`
FFS!
Others fail with similar errors or `unrecognized command-line option '-march=native'; did you mean '-mcpu=native'?`
Ok, so this is basically x86 only. Hence not blocking this PR.

Reply:
Reading the first link, I got the impression the authors only consider `x86_64` and everything else does not exist. I might be wrong. I did not know about the other programs. Unfortunately, they are really required as far as I can work it out. 
Any suggestions?

Reply:
Just accept that this is x86 only. Nothing can be done here...
Maybe open issues at the other repos and complain about it but... yeah.

Reply:
For BWA I have a workaround: https://github.com/easybuilders/easybuild-easyconfigs/pull/14073

Reply:
The workaround for ARM from [here](https://aws.amazon.com/de/blogs/publicsector/generalized-approach-benchmarking-genomics-workloads-cloud-bwa-read-aligner-graviton2/):
```bash
    sudo yum groupinstall "Development Tools" -y
    git clone https://github.com/lh3/bwa.git
    cd bwa/
    sed -i -e 's/<emmintrin.h>/"sse2neon.h"/' ksw.c
    wget https://gitlab.com/arm-hpc/packages/uploads/ca862a40906a0012de90ef7b3a98e49d/sse2neon.h
    make clean all
```
Maybe we could somehow get that into `BWA` as well so that builds there too? That would solve this whole issue I think. 
I don't have access to ARM, other than an older Raspberry Pi. so I cannot test this or help much here. 

Are the programs which fail only from existing EC like `minimap2` or are they from the extension list as well? If they are only from existing ECs, I would suggest to get them sorted which solved the problem here. 
@boegel : What are your suggestions here?